### PR TITLE
DEVPS.ENH: Includes fast correctors device

### PR DESCRIPTION
**Pull request by jquentino**

### CODE REVIEW

File: siriuspy/siriuspy/devices/pwrsupply.py [hunk 1]
```diff
@@ -34,6 +34,16 @@ class _PSDev(_Device):
         'CycleAuxParam-SP', 'CycleAuxParam-RB',
         'CycleEnbl-Mon',
     )
+    _properties_fc = (
+        'PSStatus-Mon', 'AlarmsAmp-Mon', 'OpMode-Sel', 'OpMode-Sts',
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/pwrsupply.py:38]: The PSStatus-Mon PV was removed from the current IOC version, so we can remove from here too

Comment (anacso17) [reply siriuspy/siriuspy/devices/pwrsupply.py:38]: I think we need to add 'Current-SP', 'Current-RB', 'Current-Mon', 'CurrentRef-Mon' to this tuple too

Comment (anacso17) [reply siriuspy/siriuspy/devices/pwrsupply.py:38]: also 'OpMode-Sel', 'OpMode-Sts'

Comment (anacso17) [reply siriuspy/siriuspy/devices/pwrsupply.py]: Sorry for my lack of attention, these PVs are already there...

**Review by anacso17 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**